src/braket/aws/aws_device.py

```diff
@@ -406,7 +406,7 @@ def is_available(self) -> bool:
 
         is_available_result = False
 
-        current_datetime_utc = datetime.utcnow()
+        current_datetime_utc = datetime.utcnow()  # noqa: DTZ003
```

Comment:
Why not fix this?

Reply:
There were some weird failures happening with the change. I am planning this as a functional fix later on.